src/libexpr/parser.y

```diff
@@ -359,11 +359,18 @@ string_parts_interpolated
 
 path_start
   : PATH {
-    Path path(absPath(std::string_view{$1.p, $1.l}, state->basePath.path.abs()));
+    std::string_view literal({$1.p, $1.l});
+    Path path(absPath(literal, state->basePath.path.abs()));
```

Comment:
This `absPath` is only needed in one of the new branches below, can you move it down?

Reply:
It's always needed because it canonicalizes the path.

Reply:
OK I think we need to canonicalize with respect to the accessor, but this is a bit of long-standing separate technical debt, so fine for now.